oomact/src/model/sensors/Imu.cpp

```diff
@@ -253,10 +268,10 @@ void Imu::addMeasurementErrorTerms(CalibratorI & calib, const EstConf & /*ec*/,
     addImuErrorTerms(
         calib, *this, accelerometerName, measurements_->accelerometer,
         [&, this](const Timestamp timestamp, const AccelerometerMeasurement & m){
-          auto robot = calib.getModelAt(*this, timestamp, 2, {false});
+          auto modelAt = calib.getModelAt(*this, timestamp, 2, {false});
           return boost::make_shared<ErrorTermAccelerometer>(
-              robot.getAcceleration(inertiaFrame, getParentFrame()),
-              getTransformationExpressionTo(robot, inertiaFrame).toRotationExpression().inverse(),
+              modelAt.getAcceleration(getFrame(), inertiaFrame),
```

Comment:
Was this where the error was?

Reply:
This was the first bug, the one within the Imu, discovered quite early. However, the real big problem was this https://github.com/ethz-asl/oomact/pull/32/commits/59231c71b07b4bc62675d48d631a58bd1246441a small change. Effectively this effectively lets the same CoordinateFrames (bad name; they do the automatic kinematic) be generated but in flipped order -> the parent child relation gets inverted. This is only visible when you have more then two frames in a chain - and that wasn't used before the imu would use its own sensor frame connected to the body frame (static (non trajectory) links are now also supported (so far only for the sensors but easy to extend). As I said the FrameGraphModel is very new..

Reply:
Do you have a better idea for the naming?